common/src/main/java/com/wynntils/screens/base/widgets/SearchWidget.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
```

Comment:
I presume the changes to this class were reverted?

Reply:
Yeah, probably still in the commit history..

Reply:
But you never reverted the change..?

Reply:
I can't really do that, without triggering spotless, or doing a force push. I can revert it by force-pushing main, if you want to.

Reply:
I see the problem. Too much effort, we'll have to live with this.